requirements.txt

```diff
@@ -3,6 +3,7 @@ django-ckeditor==5.6.1
 Pillow
 python-social-auth
 python-dataporten-auth
+pytz
```

Comment:
Just because PyCharm complained :p Can revert it if we think it's unnecessary.

Reply:
Also, I do realize that it's kind of unrelated to this pull request, but it was a quick add.

Reply:
I find it a bit weird that Pycharm complains about it, since IntelliJ does not. Anyways, I think we should remove it. It has been a dependency of Django for two years. And I do not think we use it outside of Django anyways.